tools/tenscan/frontend/api/general.ts

```diff
@@ -7,8 +7,8 @@ export const fetchTestnetStatus = async (): Promise<
   ResponseDataInterface<boolean>
 > => {
   return await httpRequest<ResponseDataInterface<boolean>>({
-    method: "post",
+    method: "get",
     url: pathToUrl(apiRoutes.getHealthStatus),
-    data: { jsonrpc: "2.0", method: "obscuro_health", params: [], id: 1 },
```

Comment:
I think we can get rid of this param from both functions since we're calling the HTTP endpoint at `/info/health` and not the RPC endpoint at `obscuro_health`

Reply:
removed 🗑️